bodhi/server/consumers/greenwave.py

```diff
@@ -0,0 +1,61 @@
+# Copyright © 2016-2019 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+The "greenwave handler".
+
+This module is responsible for listening for messages from greenwave.
+It then updates the policies of the build that greenwave checked.
+"""
+
+import logging
+
+import fedora_messaging
+
+from bodhi.server import initialize_db
+from bodhi.server.config import config
+from bodhi.server.models import Build
+from bodhi.server.models import Update
+from bodhi.server.util import transactional_session_maker
+
+log = logging.getLogger('bodhi')
+
+
+class GreenwaveHandler(object):
```

Comment:
Since we are Python 3 only, we don't need to subclass ```object``` anymore:

```suggestion
class GreenwaveHandler:
```